### PR TITLE
Master accexport

### PR DESCRIPTION
**Pull request by xschildw**

### CODE REVIEW

File: templates/SynapseCMK-template.json [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+				}
+        },
+	"Resources": {
+		"SynapseDeploymentPolicy": {
+			"Type": "AWS::IAM::Policy",
+			"Properties": {
+				"PolicyName": "SynapseDeploymentPolicy",
+				"PolicyDocument": {
+					"Version": "2012-10-17",
+					"Statement": [
```
Comment (zaro0508) [inline templates/SynapseCMK-template.json]: why not just give all admin permissions here?  your pretty much almost there.  also it's kind of a hassle to muck with this every time you add stuff to CF templates.

File: templates/SynapseCMK-template.json [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+								{
+									"AWS": [
+										{
+											"Fn::Join": [
+												"",
+												["arn:aws:iam::", {"Ref": "AWS::AccountId"},":root"]
+											]
+										},
+										{
+											"Fn::ImportValue": "us-east-1-accounts-AWSIAMAdminRoleArn"
```
Comment (zaro0508) [inline templates/SynapseCMK-template.json]: Again, if you plan to use CI to manage this key then you'll probably want to either change this to the CF service role or add the cf service role.

File: templates/SynapseCMK-template.json [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+			"Type": "AWS::IAM::Policy",
+			"Properties": {
+				"PolicyName": "SynapseDeploymentPolicy",
+				"PolicyDocument": {
+					"Version": "2012-10-17",
+					"Statement": [
+						{"Effect": "Allow", "Action": "s3:*", "Resource": "*"},
+						{"Effect": "Allow", "Action": "sns:*", "Resource": "*"},
+						{"Effect": "Allow", "Action": "rds:*", "Resource": "*"},
+						{"Effect": "Allow", "Action": "ec2:*", "Resource": "*"},
```
Comment (xschildw) [inline templates/SynapseCMK-template.json]: Add load balancer permissions (describe)

Comment (xschildw) [reply templates/SynapseCMK-template.json]: All admin is what we have now, we would like to reduce the surface as much as we can (first step was the services, next one is the API and/or resources).

File: templates/SynapseCMK-template.json (hunk 2 above)
Comment (xschildw) [reply templates/SynapseCMK-template.json]: Per discussion with John, no plan to include into CI.

**Pull request merged**